src/components/organisms/NavBarSchedule/utils.ts

```diff
@@ -14,12 +14,25 @@ import { WithVenueId } from "utils/id";
 import { eventEndTime, eventStartTime } from "utils/event";
 import { isTruthy } from "utils/types";
 
-export const prepareForSchedule = (
-  day: Date,
-  usersEvents: MyPersonalizedSchedule
-) => (event: WithVenueId<VenueEvent>): PersonalizedVenueEvent => {
-  const startOfEventToShow = max([eventStartTime(event), startOfDay(day)]);
-  const endOfEventToShow = min([eventEndTime(event), endOfDay(day)]);
+export interface PrepareForScheduleProps {
+  day: Date;
+  usersEvents: MyPersonalizedSchedule;
+  isForCalendarFile: boolean;
+}
```

Comment:
We can make `isForCalendarFile` an optional prop here:
```suggestion
export interface PrepareForScheduleProps {
  day: Date;
  usersEvents: MyPersonalizedSchedule;
  isForCalendarFile?: boolean;
}
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1423/commits/1346ad92bc3006ba3bbbe1f5695c2806dcc4f4c4